### ISSUE TITLE
Add learning curve module

### ISSUE THREAD

**Issue by fkunneman** [LanguageMachines/quoll#5]
A learning curve module is useful for obtaining an indication on the effect of the amount of training data on performance. 

The learning curve module should take the form of a wrapper around the Report module with train and test input. The input to learning curve can be a set of instances and labels, after which a train and test split is made and the training data is binned, or a train and test set, after which only the train data is binned. 

An implementation of the learning curve was already made in version 0, and can be consulted for inspiration.
